include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -915,7 +913,6 @@ Make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
             static void writeField(
                 ThreadParams* params,
                 const uint32_t sizePtrType,
```

Comment:
Unrelated to this PR, but why is this variable here?

Reply:
Good question, honestly I don't know. I can remove it, but then the CI would need to run again.

Reply:
It can also be done separately, was just wondering if that variable had some sense that I don't get.

Reply:
>It can also be done separately

Done: https://github.com/ComputationalRadiationPhysics/picongpu/pull/3767